src/Examples/45Example/Controllers/AccountController.cs

```diff
@@ -33,30 +36,30 @@ public ActionResult Connect(string token)
                 var activityDetails = yotiClient.GetActivityDetails(token);
                 if (activityDetails.Outcome == ActivityOutcome.Success)
                 {
-                    var yotiProfile = activityDetails.UserProfile;
+                    var profile = activityDetails.Profile;
```

Comment:
when did we decide to go for `Profile`? Also this is not backward compatible unless it it an addition.

Reply:
Yes we decided to go for `Profile`, so that the clients only have to update this one line, rather than changing all of the attributes from `GivenNames` -> `GivenNamesAttribute` etc.. It is an addition. We will remove `UserProfile` in the next major version